source/hpack.c

```diff
@@ -479,13 +222,15 @@ size_t aws_hpack_find_index(
         /* Check name-and-value first in static table */
         aws_hash_table_find(&s_static_header_reverse_lookup, header, &elem);
         if (elem) {
+            /* TODO: Maybe always set found_value to true? Who cares that the value is empty if they matched? */
```

Comment:
I read through the spec again, doesn't seem like it matters...

I'll vote for always set found_value to true this case. It seems less steps to handle for both side

Reply:
I'm leaving this TODO so that this commit can be as minimal as possible: just moving code around, no functional changes